src/app/components/ViewLife.tsx

```diff
@@ -0,0 +1,75 @@
+import { Skeleton, Descriptions, Result, Button, Divider } from 'antd';
+import { useTranslation } from 'react-i18next';
+import { useParams, useHistory } from 'react-router-dom';
+import styled from 'styled-components';
+import { useGetLifeQuery } from '../api';
+
+const StyledDescription = styled(Descriptions)`
+    .ant-descriptions-title {
+        font-size: 2em;
+        text-align: center;
+    }
+    .ant-descriptions-item-label {
+        font-weight: bold;
+        background-color: #d9d9d9;
+    }
+`;
+
+const ViewLife = () => {
+    const { t } = useTranslation('life');
+    const history = useHistory();
+    const params: { id: string } = useParams();
+    const { data, loading } = useGetLifeQuery({ variables: { id: params.id } });
+    const lifeDetails = data?.getLife;
+
+    return (
+        <Skeleton loading={loading} active>
+            {!lifeDetails && (
+                <Result
+                    extra={
+                        <Button onClick={() => history.push('/lives')} type="primary">
+                            Back to Lives
+                        </Button>
+                    }
+                    status="404"
+                    subTitle={t('life:viewLife:lifeNotFound')}
```

Comment:
keys should be separated by `.` and not by `:` You only separate the namespace and keys by `:`
So this should be `life:viewLife.lifeNotFound`

Reply:
Done